### DISCUSSION TITLE
Macros included

### DISCUSSION THREAD

**Discussion by PeteDuncanson** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#22] [General]
Existing start kit doesn't use Marcos or show them on the Rich Text Editor. As a result we've had a generation of HQ devs (and users out in the wild) totally unaware of their existence and usefulness. They should be included in the starter as its historically been the thing that everyone installs to show stuff off. As per #21 there should be a tour about what macros are, why you would need one and how to create one.

**Comment by TimGeyssens** [comment]
Ok valid idea :) are macros still used a lot? I guess indeed for adding stuff to the rte...

**Comment by TimGeyssens** [comment]
And do macros work with the new block editors?
